### PR TITLE
Beserker rav nerf

### PR DESCRIPTION
**Pull request by stanalbatross**
# About the pull request

Nerfs beserker rav somewhat.

# Explain why it's good for the game

Currently beserker rav is a bit overtuned, being able to deal large damage up close as well as quickly heal and avoid the consequences of going in close. This PR seeks to tune down their up close killing potential a bit by reducing the superslow to a slow, and makes their health-regaining abilities a bit weaker. This will make beserker a bit easier to fight and less frustrating, and add some risk to engagements. 

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
balance: increased clothesline cooldown from 14s to 16s
balance: beserker rav buffed slash will now slow instead of superslow, and its duration will be 3.5s as intended instead of 0.6s
balance: ravs can no longer heal from clothesline/eviscerate while on fire
/:cl:

### CODE REVIEW

**Comment by Pyrodes** [comment]
and now zerker goes down the unbalancable caste rabbit hole because of one needless change to dumb it down for the less-skilled players

**Review by harryob — APPROVED** [review]

**Comment by harryob** [comment]
merging as architect approved
